client/lib/site-stats-sticky-tab/store.js

```diff
@@ -1,137 +0,0 @@
-// External dependencies
```

Comment:
💥 -- if we ever wanted to pull any of this logic out into reducers, selectors, etc. it's in version control.